src/views/print.c

```diff
@@ -321,7 +321,7 @@ gboolean try_enter(dt_view_t *self)
 
   //  now check that there is at least one selected image
 
-  const int32_t imgid = dt_act_on_get_main_image();
+  const dt_imgid_t imgid = dt_act_on_get_main_image();
 
   if(imgid < 0)
```

Comment:
This seems wrong

Reply:
No, as dt_act_on_get_main_image() could return -1 before.